connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -1795,11 +1795,9 @@ export async function renderAndUpsertPageFromCache({
           parentDb.notionDatabaseId,
           [],
           true,
-          runTimestamp.toString(),
-          async () => {
-            await heartbeat();
-          }
+          runTimestamp.toString()
         );
+        await heartbeat();
```

Comment:
I don't think we want this:
- heartbeat is very quick
- the purpose of heartbeating is triggering an exception if we're running over the start to close timeout (otherwise you get a zombie process that continues running an activity that is already timed out and retried)